xarray/core/computation.py

```diff
@@ -1047,6 +1046,163 @@ def earth_mover_distance(first_samples,
         return apply_array_ufunc(func, *args, dask=dask)
 
 
+def cov(da_a, da_b, dim=None):
+    """Compute covariance between two DataArray objects along a shared dimension.
+
+    Parameters
+    ----------
+    arrays: DataArray (or Variable) objects
```

Comment:
I realize these are mostly positional, but we should probably match the parameters here to the kwargs. e.g. if someone passes `arrays=`, they're going to be confused...

Reply:
Yes, thanks for the precise control. I thought of also using arrays as parameter at one point, but guess explicit `da_a` and `da_b` is more explicit and better readable.